### ISSUE TITLE
[Documents] Error appears when trying to generate and upload new documents/quarantine orders

### ISSUE THREAD

**Issue by adinaflorea9** [hzi-braunschweig/SORMAS-Project#10985]
### Description

An error message appears when a user tries to generate and upload a document on cases, contacts, events and event participants.

### Steps to Reproduce

1. Log in the web app with an Admin+National User;
2. From the 'Configuration' directory - 'Document Templates' tab make sure to download and then upload in the system the correct types of documents for each available entity;
3. Navigate to the case page of an existing case;
4. Click on the 'Create' button from the 'Document Templates' card, check the 'Also upload' checkbox from the pop-up that opens, and select from the dropdown the available type of document;
5. Click 'Create'.

### Actual Behavior

The document is not generated and not downloaded as it should; also it is not uploaded in the UI as it should. When closing the widow  that is still open, it is visible that an error states that the document type is no correct, even though it is the template generated by the system.

```
Error while reading document variables. (de.symeda.sormas.ui.docgeneration.QuarantineOrderLayout.java:198)
de.symeda.sormas.api.docgeneneration.DocumentTemplateException: Error processing template.
	at de.symeda.sormas.backend.docgeneration.QuarantineOrderFacadeEjb.uploadDocument(QuarantineOrderFacadeEjb.java:92)
	at de.symeda.sormas.backend.docgeneration.QuarantineOrderFacadeEjb.getGeneratedDocument(QuarantineOrderFacadeEjb.java:73)
```

### Expected Behavior

- Log the actual exception for debugging (which is currently dumped by the exception handling in `QuarantineOrderFacadeEjb.uploadDocument`
- Upload of documents should be successful as long as they are the correct type of document.

### Screenshots

![documents](https://user-images.githubusercontent.com/96909231/203281971-53f7d0c1-5ebd-4c50-b495-f908ec017116.gif)

### System Details

* Device: Windows 10
* SORMAS version: 1.77.0-SNAPSHOT-(b64fb64)
* Android version/Browser: Chrome
* Server URL: local machine
* User Role: Admin+NationalUser

### Additional Information

This behavior is reproducible so far only on this exact version of SORMAS - 1.77.0-(b64fb64). At this time, it is not reproducible on testing instances where SORMAS version is 1.77.0-SNAPSHOT(3938969) or released versions.

I logged this finding as I could not determine from the latest commits which one caused the error to appear.

Logs from the local machine:
[documents logs.txt](https://github.com/hzi-braunschweig/SORMAS-Project/files/10066135/documents.logs.txt)

**Comment by StefanKock** [comment]
Discussed with @adinaflorea9 that the file upload is related to #9965 and there might be configuration missing on TEST stage, but to remove it from this finding and just focus on why the template filling does not work.

**Comment by adinaflorea9** [comment]
It seems that the issue that remained separate in this ticket was partly fixed when the changes on #9965 were merged as now on my local machine the document creation upload and the filling of the template work as expected for cases and contacts (SORMAS version 1.77.0-SNAPSHOT(af03687)).

The issue remains reproducible for events:
![image](https://user-images.githubusercontent.com/96909231/203590171-8ddbfe36-1b63-41ad-aedf-0930f8e4109e.png)

This seems to be due to the fact that the format that used to be supported for event handouts, html, is no longer among those default values. I logged a comment to #9965 as well.
